op-chain-ops/genesis/check.go

```diff
@@ -26,19 +29,68 @@ import (
+
+	LegacyETHCheckSlots = map[common.Hash]common.Hash{
+		// Bridge
+		common.Hash{31: 0x06}: common.HexToHash("0x0000000000000000000000004200000000000000000000000000000000000010"),
+		// Symbol
+		common.Hash{31: 0x04}: common.HexToHash("0x4554480000000000000000000000000000000000000000000000000000000006"),
+		// Name
+		common.Hash{31: 0x03}: common.HexToHash("0x457468657200000000000000000000000000000000000000000000000000000a"),
+		// Total supply
+		common.Hash{31: 0x02}: {},
+	}
+
+	// ExpectedStorageSlots is a map of predeploy addresses to the storage slots and values that are
+	// expected to be set in those predeploys after the migration. It does not include any predeploys
+	// that were not wiped. It also accounts for the 2 EIP-1967 storage slots in each contract.
+	// It does _not_ include L1Block. L1Block is checked separately.
+	ExpectedStorageSlots = map[common.Address]StorageCheckMap{
+		predeploys.L2CrossDomainMessengerAddr: {
+			// Slot 0x00 (0) is a combination of spacer_0_0_20, _initialized, and _initializing
+			common.Hash{}: common.HexToHash("0x0000000000000000000000010000000000000000000000000000000000000000"),
+			// Slot 0x33 (51) is _owner. Requires custom check, so set to a garbage value
+			L2XDMOwnerSlot: common.HexToHash("0xbadbadbadbad0xbadbadbadbad0xbadbadbadbad0xbadbadbadbad0xbadbad00"),
+			// Slot 0x97 (151) is _status
+			common.Hash{31: 0x97}: common.HexToHash("0x0000000000000000000000000000000000000000000000000000000000000001"),
+			// Slot 0xcc (204) is xDomainMsgSender
+			common.Hash{31: 0xcc}: common.HexToHash("0x000000000000000000000000000000000000000000000000000000000000dead"),
+			// EIP-1967 storage slots
+			AdminSlot:          common.HexToHash("0x0000000000000000000000004200000000000000000000000000000000000018"),
+			ImplementationSlot: common.HexToHash("0x000000000000000000000000c0d3c0d3c0d3c0d3c0d3c0d3c0d3c0d3c0d30007"),
+		},
+		predeploys.L2StandardBridgeAddr:             eip1967Slots(predeploys.L2StandardBridgeAddr),
+		predeploys.SequencerFeeVaultAddr:            eip1967Slots(predeploys.SequencerFeeVaultAddr),
+		predeploys.OptimismMintableERC20FactoryAddr: eip1967Slots(predeploys.OptimismMintableERC20FactoryAddr),
+		predeploys.L1BlockNumberAddr:                eip1967Slots(predeploys.L1BlockNumberAddr),
+		predeploys.GasPriceOracleAddr:               eip1967Slots(predeploys.GasPriceOracleAddr),
+		//predeploys.L1BlockAddr:                       eip1967Slots(predeploys.L1BlockAddr),
+		predeploys.L2ERC721BridgeAddr:                eip1967Slots(predeploys.L2ERC721BridgeAddr),
+		predeploys.OptimismMintableERC721FactoryAddr: eip1967Slots(predeploys.OptimismMintableERC721FactoryAddr),
+		// ProxyAdmin is not a proxy, and only has the _owner slot set.
+		predeploys.ProxyAdminAddr: {
+			// Slot 0x00 (0) is _owner. Requires custom check, so set to a garbage value
+			ProxyAdminOwnerSlot: common.HexToHash("0xbadbadbadbad0xbadbadbadbad0xbadbadbadbad0xbadbadbadbad0xbadbad00"),
```

Comment:
Same as above, why does this contain additional `0x`s inside of it?

Reply:
See above